aws/cluster/variables.tf

```diff
@@ -96,6 +96,17 @@ variable "kops-state-bucket" {
   description = "Name of the bucket in which kops stores its state (must be created prior to cluster turnup)"
 }
 
+variable "kops-addons" {
```

Comment:
Note that there are 2 types of KOPS addons, a static (manifest) ones and built-in managed ones. https://kops.sigs.k8s.io/addons/ the variable should either support both or be future proof. e.g. what about `kops-addon-manifests` or different name?

Reply:
Good point, I was checking only docu for kops version we use now (1.14.1). In order to be future-proof and match the official docu, I would go for `kops-static-addons` and add a link to the currently latest docu. https://github.com/kubernetes/kops/blob/1ae09e86a5e48e16cb885b8d001023cd1894d59e/docs/addons.md